README.md

```diff
@@ -1,4 +1,114 @@
 # Terraform VMware Cloud Foundation Automation Provider
-The official Terraform provider for VCF 9+ Automation by Broadcom
+The official Terraform provider for VMware Cloud Foundation Automation 9+ by Broadcom
 
 - This project is using [go-vcloud-director](https://github.com/vmware/go-vcloud-director) Go SDK for making API calls
+
+## Part of Terraform
+
+- Website: https://www.terraform.io
+- [Hashicorp Discuss](https://discuss.hashicorp.com/c/terraform-core/27)
+
+<img src="https://www.datocms-assets.com/2885/1629941242-logo-terraform-main.svg" width="600px">
+
+## Requirements
+
+- [Terraform](https://www.terraform.io/downloads.html)
+- [Go](https://golang.org/doc/install) 1.22 (to build the provider plugin)
+
+## Building the Provider
+
+**Note:** You *only* need to build the provider plugin if you want to *develop* it. Refer to
+[documentation](https://registry.terraform.io/providers/vmware/vcfa/latest/docs) for using it. Terraform will
+automatically download officially released binaries of this provider plugin on the first run of `terraform init`
+command.
+
+```
+$ cd ~/mydir
+$ git clone https://github.com/vmware/terraform-provider-vcfa.git
+$ cd terraform-provider-vcfa/
+$ make build
+```
+
+## Installing the Provider from source code
+
+**Note:** You *only* need to install the provider from source code if you want to test unreleased features or to develop it. Refer to
+[documentation](https://registry.terraform.io/providers/vmware/vcfa/latest/docs) for using it in a standard way. Terraform will
+automatically download officially released binaries of this provider plugin on the first run of `terraform init`
+command.
+
+```
+$ cd ~/mydir
+$ git clone https://github.com/vmware/terraform-provider-vcfa.git
+$ cd terraform-provider-vcfa/
+$ make install
+```
+
+This command will build the plugin and transfer it to
+`$HOME/.terraform.d/plugins/registry.terraform.io/vmware/vcfa/${VERSION}/${OS}_amd64/terraform-provider-vcfa_v${VERSION}`,
```

Comment:
I just noticed this "${OS}_amd64". Maybe let's use "${OS}_${ARCH}".
Or for most precise and copy-paste-able could use `$(go env GOOS)_$(go env GOARCH)`, but it's a lot..

Reply:
Changed, thanks!